backend/src/test/java/com/java/academy/gym/model/UserMessageTest.java

```diff
@@ -0,0 +1,38 @@
+package com.java.academy.gym.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+public class UserMessageTest {
+
+    @Test
+    public void shouldReturnThatUserMessagesAreEqual() {
+        Locale locale1 = new Locale("pl_PL", "polish", "Poland");
+        Locale locale2 = new Locale("pl_PL", "polish", "Poland");
```

Comment:
In integration tests this test could not pass. Why? because "@Column(unique = true)"

Reply:
Then it is a good thing that it is only a unit test :P

Reply:
@arturAR I don't understand, this is only unit test as @mprtcz said. Do you want to exclude languageCode from equals method or what do yo mean?

Reply:
yes it's just a unit test, it doesn't care about database constraints, but for this purpose more valuable is a integration test